lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/FailedEvaluatorBridge.java

```diff
@@ -55,14 +56,35 @@ public FailedEvaluatorBridge(final FailedEvaluator failedEvaluator,
     this.activeContextBridgeFactory = activeContextBridgeFactory;
   }
 
+  /**
+   * @return the Evaluator number.
+   */
   public int getNewlyRequestedEvaluatorNumber() {
     return evaluatorRequestorBridge.getEvaluatorNumber();
   }
 
+  /**
+   * @return the Evaluator requestor.
+   */
   public EvaluatorRequestorBridge getEvaluatorRequestorBridge() {
     return evaluatorRequestorBridge;
   }
 
+  /**
+   * @return the non-serializable error in bytes, may translate into a serialized C# Exception.
+   */
+  public byte[] getErrorBytes() {
```

Comment:
Then that means that the Exception originates from Java. I've added a comment in my next iteration.